docs/UserGuide.md

```diff
@@ -69,123 +75,145 @@ Format: `help`
 Switches to either `contact`, `meeting` or `deliverable` mode.
 
 Format: `switch MODE`
-* Subsequent commands will be with respect to managing the mode.
+* Subsequent commands will be with respect to the `MODE`.
```

Comment:
Also note down what happens when user switches mode (e.g list of items are displayed)